.gitignore

```diff
@@ -6,3 +6,12 @@
 /justspaces/local_settings.py
 /configs/local_settings.production.py
 /configs/local_settings.staging.py
+
+# JavaScript bundling
+.cache
+frontend/static/frontend/js/*
```

Comment:
The idea here is that we'll be compiling JS from the `frontend/src/js` directory to the `frontend/static/frontend/js` directory, the latter of which will allow Django to understand that it's a static file and serve it properly. We want to keep the compiled static files out of version control, because they're fully derived from the source files -- the source files should be the single point of truth, and all consumers of the code (dev, CI, staging/production) should compile those source files instead of retrieving compiled files from version control.